src/main/kotlin/io/github/gradlenexus/publishplugin/NexusRepositoryContainer.kt

```diff
@@ -21,7 +21,14 @@ import org.gradle.api.Action
 import org.gradle.api.NamedDomainObjectContainer
 
 interface NexusRepositoryContainer : NamedDomainObjectContainer<NexusRepository> {
-    fun sonatype(): NexusRepository
-    fun sonatype(closure: Closure<*>): NexusRepository
-    fun sonatype(action: Action<in NexusRepository>): NexusRepository
+    @Suppress("unused")
+    val s01 get() = NexusHost.S01
```

Comment:
Should there be `oss()` property as well? It does not sound fair to have one accessor only :)

Reply:
The old server it used by default, so for consistency maybe there could be `oss()`, but in the majority of cases is would not be used. In the remaining cases - some conditional behavior configured in code in some other plugin, probably it should be possible to refer just to the enum.

Reply:
I think we should add `oss` for people who want to be explicit, e.g. when having to use different hosts in different projects.